content/get-started/getting-started-with-git/configuring-git-to-handle-line-endings.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Configuring Git to handle line endings
+title: About line endings
```

Comment:
👋 @khattakdev Thanks for opening a PR with this fix 🎉  Instead of changing the titles on these articles, can you add your update after the front matter (the `---` on line 16 for this article)?

So you'll need to: 
- remove your changes to the title of each article
- add your new header (for this one it's `## About line endings`) to a new line above the first paragraph (for this article that starts on line 17)

Let me know if you need a hand or additional clarification!